### PR TITLE
Improve packaging and debugging

### PR DESCRIPTION
**Pull request by mimcz**
* Change structure a bit to reduce `ipk` bunde size
* Use ares CLI with npm scripts to make life easier
* Some minor app icon designg improvements
* Add await DRM service script (waits for DRM service before we start player)
* Add some more linting to the project

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+  bitmovin.player.core.Player.addModule(window.bitmovin.player['container-mp4'].default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player['container-ts'].default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.mserenderer.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.abr.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.drm.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.xml.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.dash.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.hls.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.style.default);
+  bitmovin.player.core.Player.addModule(window.bitmovin.player.tizen.default);
```
Comment (mukulmishra18) [inline src/js/main.js]: Are you sure we need `tizen` module in webos demo?

Also, I think you should include `webos` module.

**Review by mukulmishra18 — CHANGES_REQUESTED** [review]

File: src/js/main.js (hunk 1 above)
Comment (mimcz) [reply src/js/main.js]: @mukulmishra18 Thx for spotting this, not sure how it got here. Any idea @jdmartins ?

Comment (mimcz) [reply src/js/main.js]: Changed to webos

Comment (jdgo-mars) [reply src/js/main.js]: yes it should be changed

**Review by jdgo-mars — APPROVED** [review]
LGTM, nice workflow improvement

**Review by mukulmishra18 — APPROVED** [review]
LGTM 👍

**Pull request merged**